docs/modules/reference/nav.adoc

```diff
@@ -35,6 +35,7 @@
 ** xref:service-assurance/monitors/CiscoIpSlaMonitor.adoc[]
 ** xref:service-assurance/monitors/CiscoPingMibMonitor.adoc[]
 ** xref:service-assurance/monitors/CitrixMonitor.adoc[]
+** xref:service-assurance/monitors/DeviceConfiMonitor.adoc[]
```

Comment:
verifying spelling ... the file is DeviceConfiMonitor.adoc, but chapter title is "DeviceConfigMonitor" (see comment line 2 in DeviceConfiMonitor.adoc file.